### ISSUE TITLE
Can not init Perplexity()

### ISSUE THREAD

**Issue by phamxtien** [nathanrchn/perplexityai#56]
Today it returns error on perplexty = Perplexity()

It raise error on _get_sid() function
self.session.get(url=f"https://www.perplexity.ai/socket.io/?EIO=4&transport=polling&t={self.t}") >> Error 403

**Comment by Amm1rr** [comment]
No, I searched through many repositories and codes, but the core of all of them is from this repository, and nobody found a solution. I think the problem is with Cloudflare.
Welcome to any suggestions.

**Comment by phamxtien** [comment]
Before, I just opened the browser and went to perplexity.ai then it worked fine but now it doesn't

**Comment by Amm1rr** [comment]
I move on to the next step, but now I'm encountering an issue opening the WebSocket connection:
`WebSocket error: Handshake status 403 Forbidden -+-+-`
